### ISSUE TITLE
Intervention module does not work in No Dynamic Segmentation mode

### ISSUE THREAD

**Issue by Chatewgne** [GeotrekCE/Geotrek-admin#3285]
Interventions are linked to Path by default, which are unavailable in NDS mode (`TREKKING_TOPOLOGY_ENABLED=False`)

![image](https://user-images.githubusercontent.com/26015573/194350499-cfccffa3-8963-49d4-b2b5-c143aabeb1c4.png)

It is impossible to create an Intervention in NDS mode at the moment (throws exceptions).

We should assess what to do with this module 
- Disable it entirely in NDS mode ?
- Change Intervention default target to something else such as signage ?

**Comment by babastienne** [comment]
My opinion is that in NDS mode we should this module, as well as other features that still are "visible" to the users but not completely working (example in the screenshot below, we display the status options when visiting a trek even if we should not be able to use this module)

![image](https://user-images.githubusercontent.com/16348048/194367477-0b6dd272-8896-4628-8787-d2496a5e52ff.png)


There is some work to do in geotrek in order the hide some features in NDS mode that should not be visible. the intervention module is one of them in my opinion.

**Comment by camillemonchicourt** [comment]
Yes, these modules should not be available in NDS mode, as they can't work without paths and topologies.
